week_4_analytics_engineering/README.md

```diff
@@ -116,6 +116,7 @@ Did you take notes? You can share them here.
 
 * [Notes by Alvaro Navas](https://github.com/ziritrion/dataeng-zoomcamp/blob/main/notes/4_analytics.md)
 * [Sandy's DE learning blog](https://learningdataengineering540969211.wordpress.com/2022/02/17/week-4-setting-up-dbt-cloud-with-bigquery/)
+* [Karen's notes](https://github.com/Karenzhang7717/analytics-engineering-with-dbt)
```

Comment:
Are you sure this is the right link? It goes to your repository, no notes in there